src/components/VideoCard/index.jsx

```diff
@@ -20,6 +33,17 @@ const VideoCard = (props) => {
         <Description dark={globalState.state.darkMode ? 'true' : undefined}>
           {props.video.snippet.description}
         </Description>
+        {props.favorites === true ? (
+          <FavoritesIcon>
+            <FavoriteIcon
+              data-testid="favorite-icon"
+              color="primary"
+              onClick={removeFromFavorite}
+            />{' '}
+          </FavoritesIcon>
+        ) : (
+          <></>
+        )}
```

Comment:
This should simplify your code and work the same:
```suggestion
        {props.favorites && (
          <FavoritesIcon>
            <FavoriteIcon
              data-testid="favorite-icon"
              color="primary"
              onClick={removeFromFavorite}
            />{' '}
          </FavoritesIcon>
        )}
```